frontend/src/scenes/dashboard/dashboardLogic.ts

```diff
@@ -26,6 +26,7 @@ import { teamLogic } from '../teamLogic'
 import { urls } from 'scenes/urls'
 import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
 import { userLogic } from 'scenes/userLogic'
+import { withDataFromDashboardContext } from 'scenes/insights/utils/dataStorage'
```

Comment:
I've learned a while ago that `data` is a bad word, and the second-worst variable name, after `data2`. It basically says nothing, since everything is data... but what kind of data? "Field" or "record" is a similarly ambiguous term, but already hints at the source or type of data.

So, can this be clearer? E.g. `applyDashboardFieldsToInsight` or `mergeWithDashboardTile`? I don't think we need to bring in another term called "context" here 🤔